cmd/snap/cmd_quota_test.go

```diff
@@ -154,3 +168,47 @@ func (s *validateSuite) TestRemoveQuotaGroup(c *check.C) {
 	c.Check(s.Stderr(), check.Equals, "")
 	c.Check(s.Stdout(), check.Equals, "")
 }
+
+func (s *quotaSuite) TestGetAllQuotaGroups(c *check.C) {
+	restore := main.MockIsStdinTTY(true)
+	defer restore()
+
+	s.RedirectClientToTestServer(makeFakeGetQuotaGroupsHandler(c,
+		`{"type": "sync", "status-code": 200, "result": [
+			{"group-name":"aaa","subgroups":["ccc","ddd"],"parent":"zzz","max-memory":1000},
+			{"group-name":"ddd","parent":"aaa","max-memory":400},
+			{"group-name":"bbb","parent":"zzz","max-memory":1000},
+			{"group-name":"yyy","max-memory":1000},
```

Comment:
We should probably have at least one group name be longer than `Quota` to ensure that output is formatted correctly.

Reply:
Done